tests/e2e/Services/FunctionsSchedule/FunctionsScheduleTest.php

```diff
@@ -137,6 +137,29 @@ public function testCreateScheduledAtExecution(): void
             $this->assertStringContainsString('user-is-' . $this->getUser()['$id'], $execution['body']['logs']);
             $this->assertStringContainsString('jwt-is-valid', $execution['body']['logs']);
             $this->assertGreaterThan(0, $execution['body']['duration']);
+
+            $logs = explode("\n", $execution['body']['logs']);
+            $scheduleDelay = null;
+            $scheduledAt = null;
+            $executedAt = null;
+
+            foreach ($logs as $line) {
+                $line = trim($line);
+
+                if (str_starts_with($line, 'schedule-delay-is-')) {
+                    $scheduleDelay = (int) substr($line, strlen('schedule-delay-is-'));
+                } elseif (str_starts_with($line, 'scheduled-at-is-')) {
+                    $scheduledAt = substr($line, strlen('scheduled-at-is-'));
+                } elseif (str_starts_with($line, 'executed-at-is-')) {
+                    $executedAt = substr($line, strlen('executed-at-is-'));
+                }
+            }
```

Comment:
Instead of parsing, I would write tests like these:

1. Read request headers, ensure there are 3 headers
1.1. scheduled and executed at header to be valid datetime (parsing with datetine will not be false)
2. build strings in test, how function would build it, using value from request headers
2. Assert logs contain exact string, somethling like `"executed-at-is-$executedAt"`